app/views/state_file/questions/nj_homeowner_property_tax/edit.html.erb

```diff
@@ -12,7 +12,7 @@
     </div>
 
     <div class="reveal">
-      <p><a href="#" class="reveal__link"><%= t('.helper_heading') %></a></p>
+      <p><button class="reveal__button"><%= t('.helper_heading') %></button></p>
```

Comment:
Do we need to add an aria-expanded value to all of these reveal buttons? Or is that handled somewhere else?

Reply:
@mmazanec22 `app/assets/javascripts/honeycrisp.js` has code for this in `revealer` part of the code where it sets `aria-expanded` to `false`, but if you're not seeing that get set, please let me know! I can double check this on Monday when I'm back

Reply:
It does appear to be working! Nice, good to know we don't need to manually add anything